pkg/libvmi/storage.go

```diff
@@ -219,15 +234,20 @@ func newVolume(name string) v1.Volume {
 	return v1.Volume{Name: name}
 }
 
-func newContainerVolume(name, image string) v1.Volume {
-	return v1.Volume{
+func newContainerVolume(name, image string, imagePullPolicy ...k8sv1.PullPolicy) v1.Volume {
```

Comment:
Can you pass `imagePullPolicy` as a string instead and check for empty string?

Reply:
okay